woocommerce/woocommerce-admin
# Display CES modal once per action per store

Fixes #5338 

The CES modal should only be displayed once per action per store. This is done using an option value (`woocommerce_ces_shown_for_actions`) which contains an array of actions that have had the CES modal shown. The JS that hooks in to the "Publish" button on the "Add new product" page is not added if the option contains the current action. The CES tracks component is also not shown if the option contains the current action.

### Detailed test instructions:

- Add and publish a new product. The modal should be displayed.
- Refresh the page. The modal should not be displayed.
- Add and publish a new product. The modal should not be displayed.
- Delete the `woocommerce_ces_shown_for_actions` option then add a new product. The modal should be displayed again.


### Changelog Note:

Dev: Display CES modal once per action per store
